### ISSUE TITLE
很好的项目

### ISSUE THREAD

**Issue by Dikea** [zqwerty/CVAE#2]
您好，最近学习对话领域的cvae模型，看到了您的代码，感觉非常棒！
请问您这个项目是比较完整的吗，模型的效果是还不错的吗？
看您是采用了微博的对话语料，请问里面的 train_ref 数据您是如何构造的呢？

**Comment by zqwerty** [comment]
效果其实不太好:(, cvae 做语言生成会有KL消失的问题，decoder太强了，z的控制不够

**Comment by Dikea** [comment]
> 效果其实不太好:(, cvae 做语言生成会有KL消失的问题，decoder太强了，z的控制不够

我也写了一个，参考了一些kgcvae原作者的代码，kld loss的确很小，不过没有消失不见，不知道kld loss 是如何范围，有兴趣，可以一起讨论下，我微信 2842721963 。
```
5123 INFO:tensorflow:loss = 9.192852, step = 165200 (99.730 sec)
5124 INFO:tensorflow:gen_loss = 3.7730725, bow_loss = 5.4108863, kld_loss = 0.008893368 (99.730 sec)
5125 INFO:tensorflow:global_step/sec: 1.00251
5126 INFO:tensorflow:loss = 9.197446, step = 165300 (99.750 sec)
5127 INFO:tensorflow:gen_loss = 3.7973697, bow_loss = 5.3898897, kld_loss = 0.010185994 (99.750 sec)
5128 INFO:tensorflow:global_step/sec: 1.0022
5129 INFO:tensorflow:loss = 9.22631, step = 165400 (99.781 sec)
5130 INFO:tensorflow:gen_loss = 3.813841, bow_loss = 5.4037313, kld_loss = 0.008737763 (99.781 sec)
5131 INFO:tensorflow:global_step/sec: 1.00264
5132 INFO:tensorflow:loss = 9.134821, step = 165500 (99.737 sec)
5133 INFO:tensorflow:gen_loss = 3.8026948, bow_loss = 5.3224607, kld_loss = 0.009665117 (99.737 sec)
5134 INFO:tensorflow:global_step/sec: 1.00301
5135 INFO:tensorflow:loss = 9.044414, step = 165600 (99.700 sec)
5136 INFO:tensorflow:gen_loss = 3.7366362, bow_loss = 5.2980556, kld_loss = 0.009721422 (99.700 sec)
```
